errors/Readme.md

```diff
@@ -697,15 +233,15 @@ More issues for init command failures are covered here: https://prisma-specs.net
 
 ### Generate
 
-Generate command relays the following errors from the SDK: `P5000`, `P5001`, `P5002`
+!TODO
```

Comment:
This content was just wrong and is actually undefined?

Reply:
Those are schema errors again. We don't return those at the moment. And Generate will very likely return much more errors than just generate. I am unsure about the value of listing those error codes at all. It is very unlikely that a manually compiled list is right at all.